gateway/src/event.rs

```diff
@@ -134,6 +134,129 @@ bitflags! {
     }
 }
 
+impl EventTypeFlags {
+    /// All [`EventTypeFlags`] in [`Intents::GUILDS`].
+    ///
+    /// [`Intents::GUILDS`]: twilight-gateway::Intents::GUILDS
```

Comment:
These links don't actually work despite CI passing. It needs to be in underscore form.

Reply:
underscore form didn't work either :shrug:, changed to `crate`